model/database/registry.yaml

```diff
@@ -26,9 +26,9 @@ groups:
           The name of the database, fully qualified within the server address and port.
         note: >
           If a database system has multiple namespace components, they SHOULD be concatenated
-          (potentially using database system specific conventions) from most general to most
-          specific namespace component, and more specific namespaces SHOULD NOT be captured without
-          the more general namespaces, to ensure that "startswith" queries for the more general namespaces will be valid.
+          from the most general to the most specific namespace component,
+          using `|` as a separator between the components.
+          Any missing components (and their associated separators) SHOULD be omitted.
```

Comment:
I intentionally removed this:

> more specific namespaces SHOULD NOT be captured without the more general namespaces, to ensure that "startswith" queries for the more general namespaces will be valid

since it seems better to capture something than nothing.